src/middlewared/middlewared/plugins/jail.py

```diff
@@ -245,8 +245,11 @@ def fstab(self, jail, options):
         dump = options["dump"]
         _pass = options["_pass"]
 
-        iocage.fstab(action, source, destination, fstype, fsoptions, dump,
-                     _pass)
+        _list = iocage.fstab(action, source, destination, fstype, fsoptions,
+                             dump, _pass)
+
+        if action == "list":
+            return _list
```

Comment:
What kind of actions could be there?

i think it would be useful to add it to the schema, e.g.

`Str("source", enum=["LIST", "ADD"])`

ps: note the uppercase, its not written but we are trying to use uppercase for all enums for consistency =P